src/common/arrstr.cpp

```diff
@@ -573,3 +579,165 @@ wxArrayString wxSplit(const wxString& str, const wxChar sep, const wxChar escape
 
     return ret;
 }
+
+
+namespace
+{
+    wxRegEx naturalNumeric(wxS("[0-9.]+"));
```

Comment:
Hardcoding decimal separator may not work the same in the locales that do not use decimal point but use a decimal comma. Not sure how this is supposed to be handled in the natural sort algorithms, e.g., what `StrCmpLogicalW()` does...

TBH, I am also not a fan of creating global instances of (wxWidgets) classes like this but right now cannot think of a simple better solution....

Reply:
Oh no, actually it won't work, because 0.3 won't be the same as .3

Reply:
And, StrCmpLogicalW() doesn't seem to handle decimal points either.  Maybe it's better not to bother.